container_registry/container_analysis/samples.py

```diff
@@ -0,0 +1,343 @@
+#!/bin/python
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from time import sleep
+
+from google.api_core.exceptions import AlreadyExists
+from google.cloud.devtools.containeranalysis_v1alpha1 \
+    import container_analysis_client
+from google.cloud.devtools.containeranalysis_v1alpha1.proto \
+    import containeranalysis_pb2
+from google.cloud.devtools.containeranalysis_v1alpha1.proto \
+    import package_vulnerability_pb2
+from google.cloud.pubsub import SubscriberClient
+
+
+# [START create_note]
+def create_note(note_id, project_id):
+    """Creates and returns a new note
```

Comment:
We usually don't put docstrings this comprehensive into the samples as the snippets will typically be surfaced in the documentation where there is sufficient context to know what is going on.

Reply:
Ok, should I just remove the param/return/raises to make it simpler? Or should I move the docstrings outside the region tags?

Reply:
> Ok, should I just remove the param/return/raises to make it simpler?
Yup, I'd remove them.